### DISCUSSION TITLE
Mistakely remove my account from github

### DISCUSSION THREAD

**Discussion by haseebhub** [community/community#110290] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

By mistake I have removed my github account which consist of all my repostries.Before doing this I have change password for my account associated with that,but when I sign in it always shows incorrect username or password.But have link to my github account and it is showing.

**Comment by grandmas-favorite** [comment]
Thanks for posting in the GitHub Community, @haseebhub!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Actions category is solely related to conversations around the Actions product. This question should be in the General category. I've gone ahead and moved it for you. Good luck!
